src/Dashboard/Data/FunctionInstanceSnapshot.cs

```diff
@@ -17,7 +17,7 @@ public class FunctionInstanceSnapshot
 
         public Guid Id { get; set; }
 
-        public Guid HostInstanceId { get; set; }
```

Comment:
As far as testing goes, I believe the host instance Id is primarily used for host specific messaging from Dashboard (e.g. host "Abort" for runaway functions, etc.). See https://github.com/Azure/azure-webjobs-sdk/issues/770 for details.